packages/utils/src/internals/cheerio.ts

```diff
@@ -5,6 +5,7 @@ import * as cheerio from 'cheerio';
 import { tryAbsoluteURL } from './extract-urls';
 
 export type CheerioRoot = ReturnType<typeof load>;
```

Comment:
let's deprecate this one since its a duplication of `CheerioAPI`

```suggestion
/** @deprecated use CheerioAPI instead */
export type CheerioRoot = ReturnType<typeof load>;
```